frontend/src/components/RecipePage.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import "./RecipePage.css";
+import Navbar from "./Navbar";
+import { Link } from "react-router-dom";
+
+const RecipePage = (props) => {
+  const [newIngredient, setNewIngredient] = useState("");
+  const [newInstruction, setNewInstruction] = useState("");
+
+  return (
+    <div>
+      <head>
+        <title>Angela's Kitchen</title>
+        <meta charset="utf-8" />
+        <link rel="stylesheet" href="recipe.css" />
+      </head>
+      <body>
+        <Navbar />
+        <div class="intro-container">
+          <div class="intro-header">
+            <h1 id="recipeName">{props.recipeName}</h1>
+            <p id="text">{props.description}</p>
+          </div>
+          <img id="image" src={props.imageSrc} />
+        </div>
+        <section id="ingredients">
+          <h2 class="heading">Ingredients You'll Need</h2>
+          <ul id="ingredient-list">
+            {props.ingredientList.map((ingredient) => (
+              <li>{ingredient}</li>
+            ))}
+          </ul>
+        </section>
+        <section class="form">
+          <h2>Make It Your Own!</h2>
+          <label>Ingredient: </label>
+          <input
+            id="newIngredient"
+            className="form-element"
+            placeholder="Ex: 2 cups spinach"
+            value={newIngredient}
+            onChange={(e) => {
+              setNewIngredient(e.target.value);
+            }}
+            type="text"
+          />
+          <br />
+          <br />
+          <button onclick="addIngredient()">Add Ingredient</button>
```

Comment:
It looks like your missing your addIngredient() and addStep() function. These functions should be connected to your PUT requests that you created in Milestone 4!